netbox/netbox/authentication.py

```diff
@@ -163,8 +170,15 @@ def __new__(cls, *args, **kwargs):
                 "Required parameter AUTH_LDAP_SERVER_URI is missing from ldap_config.py."
             )
 
-        # Create a new instance of django-auth-ldap's LDAPBackend
-        obj = LDAPBackend_()
+        # Create a new instance of django-auth-ldap's LDAPBackend with our own ObjectPermissions
+        class NBLDAPBackend(ObjectPermissionMixin, LDAPBackend_):
+            def get_permission_filter(self, user_obj):
+                permission_filter = Q(users=user_obj) | Q(groups__user=user_obj)
```

Comment:
Might be better to invoke `super()` to ensure we don't miss this method during any future changes to the default filter.

```suggestion
                permission_filter = super().get_permission_filter(user_obj)
```